salt/states/powerdns.py

```diff
@@ -0,0 +1,118 @@
+"""
+Create, modify and delete PowerDNS zone
+"""
+
+import logging
+
+from salt.exceptions import SaltInvocationError
+
+log = logging.getLogger(__name__)
+
```

Comment:
```suggestion
def __virtual__():
    return "powerdns.manage_zone" in __salt__
```
The state module should not load of the execution module wasn't loaded.

Reply:
Fixed